### ISSUE TITLE
[Feature] New JFXButton type (Outlined Button)

### ISSUE THREAD

**Issue by skmedix** [jfoenixadmin/JFoenix#911]
https://material.io/design/components/buttons.html#outlined-button

**Comment by jfoenixadmin** [comment]
@skmedix the outlined button can be done through css by using a Flat JFXButton with a border color.

**Comment by skmedix** [comment]
I agree, technically there's not difference, but we should provide that style too, especially when it's official button type.
https://material.io/design/components/buttons.html#usage
